docs/io/unified.rst

```diff
@@ -230,6 +230,130 @@ At this time, the ``meta`` attribute of the :class:`~astropy.table.Table` class
 is simply an ordered dictionary and does not fully represent the structure of a
 FITS header (for example, keyword comments are dropped).
 
+Mixin Columns
+^^^^^^^^^^^^^^
+
+Starting with astropy 3.0 it is possible to store not only standard
+`~astropy.table.Column` objects to a FITS table HDU but also the following
+:ref:`mixin_columns`:
+
+-`astropy.time.Time`
+-`astropy.units.Quantity`
+
+Consequently, the ability to convert FITS columns which are representations of
+a ``mixin column`` into that class, while storing in an `~astropy.table.Table`
+has been provided.
+
+In general a mixin column may contain multiple data components as well as
+object attributes beyond the standard Column attributes like
+``format`` or ``description``. Abiding by the rules set by the FITS standard,
+requires mapping of these data components and object attributes to the
+appropriate FITS specification columns and keywords.
+
+Thus, a well defined protocol has been developed to allow the storage of mixin
+columns in FITS while still maintaining considerable ``round-tripping``.
+
+* Quantity
```

Comment:
Did you also write the Quantity docs? I thought that was a different PR? Maybe something went wrong when rebasing?